client/state/analytics/actions.js

```diff
@@ -101,3 +101,36 @@ export const recordPageView = ( url, title, service ) => ( {
 
 export const recordGooglePageView = ( url, title ) =>
 	recordPageView( url, title, 'ga' );
+
+export const recordRegisterDomainEvent = ( eventType, ...args ) => ( dispatch ) => {
+	const registerDomainEventHandlers = {
+		addDomainButtonClick: ( domainName, section ) => {
```

Comment:
It may be better to move these to exported functions somewhere like `state/domains/actions.js`, since these are specifically tied to how domains work.